src/Avalonia.Visuals/Media/Transform.cs

```diff
@@ -28,12 +28,31 @@ static Transform()
         /// </summary>
         public abstract Matrix Value { get; }
 
+        /// <summary>
+        /// Parses a <see cref="Transform"/> string.
+        /// </summary>
+        /// <param name="s">The string.</param>
+        /// <returns>The <see cref="Transform"/>.</returns>
+        public static Transform Parse(string s)
+        {
+            return new MatrixTransform(Matrix.Parse(s));
```

Comment:
Perhaps we could add some docs for the format of this matrix transform string? :)